packages/expo-modules-autolinking/src/types.ts

```diff
@@ -59,6 +59,12 @@ export interface RawExpoModuleConfig {
     /**
      * Names of Swift native modules classes to put to the generated modules provider file.
      */
+    modules?: string[];
+
+    /**
+     * Names of Swift native modules classes to put to the generated modules provider file.
+     * @deprecated Deprecated in favor of `modules`.
+     */
```

Comment:
Add removal annotation - I would rather be clear about our intention here, like: `will be removed in first major release`?

Reply:
Hmm, I wouldn't explicitly say when because it's hard to say 😄 Obviously we can't remove it right now, but if we do this after next SDK release, then people may have problems if they use older versions of the package but new autolinking. There is no cost to keep it as long as we think it's useful to keep backwards compatibility 🤷‍♂️

Reply:
Additional note: we can't even rename it in existing `expo-module.config.json` files, because that would also require using the new version of autolinking. This thing is very sensitive unfortunately.